lib/bank_accounts/account.rb

```diff
@@ -0,0 +1,36 @@
+module BankAccounts
+  require "pry"
+
+  class Account
+    attr_reader :id, :balance
+
+    def initialize(id, initial_balance)
+      @id = id
+
+      # raises error if initial_balance amount is negative
+      # sets initial_balance to balance
+      if initial_balance < 0
+        raise ArgumentError.new("You need money to open this account. Please try again.")
+      else
+        @balance = initial_balance
```

Comment:
Nice use of `@balance` as your instance variable so you didn't have to write another method to return this